test.properties

```diff
@@ -5610,6 +5616,12 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
     # Frontend Infra
     #
 
+    playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][frontend]=\
```

Comment:
@dsanz It seems that this is what we need to include PW test in the Testray report when runnint `ci:test:frontend`
We might need to add `client-extensions-web`

https://testray-old.liferay.com/reports/production/logs/2024-07/test-1-37/test-portal-acceptance-pullrequest(master)/17339/playwright-js-tomcat90-mysql57-jdk8/0/0/playwright-report/index.html

![image](https://github.com/liferay-frontend/liferay-portal/assets/132653342/e3ec4039-8376-49b2-a950-0ee82870c786)

Reply:
makes sense. We added it already for relevant [in the "local" test.properties](https://github.com/liferay/liferay-portal/blob/9b19e94893a957c85530c83d0ee15e8434000456/modules/apps/frontend-data-set/test.properties#L5-L7). Idk what's the _right_ place if that thing exists.